v7/gallery_directive/README.md

```diff
@@ -5,4 +5,44 @@ Usage::
     .. gallery:: demo
 
 This should embed the gallery found in galleries/demo in your post.
-Keep in mind that this is a horrible, horrible hack.
+Keep in mind that this is sort of a hack.
+
+Caveats:
+
+* The styling of the displayed gallery is meant to sort-of-work
+with the bootstrap-based themes
+
+* It will **not** have the fancy rows/columns layout of the regular
+gallery in bootstrap, either.
+
+* It will look bad in most other themes. But you can customize it by
+creating your own gallery_fancy.tmpl. Here's the one that comes with
```

Comment:
git add templates/; git commit -am "Forgot gallery_fancy templates"

(also, `gallery_directive.tmpl` might be a better name)